### DISCUSSION TITLE
Vecchio forum: come cercare i vecchi post/contenuti

### DISCUSSION THREAD

**Discussion by indomus** [indomus/forum#16] [GENERALE - Discussioni generiche]
Fino al giorno dello switch-off della piattaforma Spectum che ospitava il vecchio forum (spegnimento previsto per i prossimi mesi), i contenuti sono ancora consultabili puntando direttamente ai link dei singoli post.

Per trovarli è sufficiente cercare su Google [in questo modo](https://www.google.com/search?q=chiave_di_ricerca+indomus+site%3Aspectrum.chat&rlz=1C5CHFA_enIT945IT945&oq=chiave_di_ricerca+indomus+site%3Aspectrum.chat&aqs=chrome..69i57.462j0j7&sourceid=chrome&ie=UTF-8), sostituendo "chiave_di_ricerca" con la chiave d'interesse.

Il Crew

**Comment by 50494554524F** [comment]
visto che non si può più scrivere sul vecchio sfrutto questo 3d per ringraziarvi della dritta dei giorni scorsi... preso le valvole tado a 40€ proprio adesso

**Reply by xyzmauro** [reply]
Ciao, se può essere utile ho provato a entrare sul mio profilo su Spectrum e nella voce "Post" ci sono i thread che ho aperto e nella voce "Activity" i thread in cui ho interagito sul vecchio forum di inDomus, quindi tutti recuperabili

**Reply by 50494554524F** [reply]
> visto che non si può più scrivere sul vecchio sfrutto questo 3d per ringraziarvi della dritta dei giorni scorsi... preso le valvole tado a 40€ proprio adesso

triplico i complimenti. se non son rotte, zero rumore, ma proprio zero.

**Reply by indomus** [reply]
sì vero, son davvero valide.

**Comment by poliant** [comment]
è possibile rispondere ai post sul vecchio forum?

**Reply by indomus** [reply]
nope
